core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/DataRowApi.kt

```diff
@@ -74,17 +74,106 @@ public operator fun AnyRow.contains(column: KProperty<*>): Boolean = containsKey
 
 // endregion
 
+/**
+ * Calculates the difference between the result of a row expression computed on the current and previous DataRow.
+ *
+ * @return [firstRowValue] for the first row; difference between expression computed for current and previous row for the following rows
+ */
+internal interface DiffDocs
+
+/**
+ * Calculates the difference between the result of a row expression computed on the current and previous DataRow.
+ *
+ * @return null for the first row; difference between expression computed for current and previous row for the following rows
+ */
+internal interface DiffOrNullDocs
+
+@OptIn(ExperimentalTypeInference::class)
+@OverloadResolutionByLambdaReturnType
+/**
+ * {@include [DiffDocs]}
```

Comment:
The inline "{}" comment brackets here aren't needed explicitly, since the `@include` statement is the only thing in the comment. But, it also does no harm to have them anyways :). Great to see you're using the processor!

Reply:
I see. I just found an example and copy-pasted it with changes. Thanks for the info, i fixed it